test/kubernetes/e2e/features/upgrade/testdata/manifests/zero-downtime-upgrade.yaml

```diff
@@ -0,0 +1,24 @@
+kubeGateway:
+  enabled: true
+  # Enable the probes to ensure zero downtime
+  gatewayParameters:
+    glooGateway:
+      service:
+        extraLabels:
+          new-service-label-key: new-service-label-val
+      podTemplate:
+        terminationGracePeriodSeconds: 5
+        gracefulShutdown:
+          enabled: true
+          sleepTimeSeconds: 2
+        probes: true
+        customLivenessProbe:
+          exec:
+            command:
+            - wget
+            - -O
+            - /dev/null
+            - 127.0.0.1:19000/server_info
+          initialDelaySeconds: 3
+          periodSeconds: 10
+          failureThreshold: 3
```

Comment:
If these are recommended settings, should we put them into our common-recommendations (https://github.com/solo-io/gloo/blob/main/test/kubernetes/e2e/tests/manifests/common-recommendations.yaml) that all tests inherit? 

I'm thinking of it more as a way to document best practices, I think we could leave this test as is, as the configuration that is relevant to this test is still nicely captured in this file